api/api/clubs/urls.py

```diff
@@ -4,5 +4,8 @@
 
 urlpatterns = [
     path('', views.ListClubsView.as_view()),
-    path('<int:pk>/', views.UpdateClubsView.as_view())
+    path('<int:pk>/', views.UpdateClubsView.as_view()),
+    path('<int:pk>/registration-requests/', views.CreateClubRegistrationRequestView.as_view()),
+    path('<int:pk_1>/registration-requests/<int:pk_2>/',
```

Comment:
let's give pk's more readable names:
for example: pk_1 can be -> club_id, pk_2 can be registration_id